composer.json

```diff
@@ -109,6 +109,7 @@
         "amphp/http-tunnel": "^1.0",
         "async-aws/ses": "^1.0",
         "async-aws/sqs": "^1.0",
+        "async-aws/sns": "^1.0",
```

Comment:
I would name it `amazon-sns` like `amazon-sqs`, I'm pretty sure that AWS provide (or will provide) several way to push messages
Same for the all the namespaces